blint/config.py

```diff
@@ -1303,6 +1305,8 @@ class BlintOptions:
     src_dir_boms: List = field(default_factory=list)
     src_dir_image: List = field(default_factory=list)
     stdout_mode: bool = False
+    use_blintdb: bool = False
+    blintdb_home: str = None
```

Comment:
@aryan-rajoria Setting to `None` immediately renders the type of `str` incorrect. It's fine to set a default of `""` instead - an empty string is still falsey, so it achieves the same purpose unless you check somewhere that the value literally _is_ `None`. We use `None` in function definitions instead of types whose value can be changed after creation (aka mutable types - like lists, dictionaries, and sets). That's also why the list types in this class are set to `field(default_factory=list)` as a default value rather than `[]`. We don't have to worry about all that with strings, which are immutable.